homeassistant/components/binary_sensor/strings.json

```diff
@@ -119,6 +123,10 @@
       "off": "[%key:component::binary_sensor::state::battery::off%]",
       "on": "Hot"
     },
+    "light": {
+      "off": "No light",
+      "on": "Light detected"
```

Comment:
I think « on/off » is good for lights

Reply:
I used the strings from  the developer documentation: https://developers.home-assistant.io/docs/core/entity/binary-sensor/ for the missing ones.
The existing ones were already consistent with the same documentation.